pkg/standalone-utils/contracts/CallAndRevert.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import { Address } from "@openzeppelin/contracts/utils/Address.sol";
+
+import { RevertCodec } from "@balancer-labs/v3-solidity-utils/contracts/helpers/RevertCodec.sol";
+
+contract CallAndRevert {
```

Comment:
maybe we can make a library out of this? There's no state to save or to access, so a library would seem better suited. Agree about making it a standalone module.

Reply:
Ah this is to reuse the hook right? Then I guess it's fine as it is.